{
	"schema_version": "1.6.0",
	"id": "USN-2671-1",
	"modified": "2015-07-09T11:24:41.454586Z",
	"published": "2015-07-09T11:24:41.454586Z",
	"related": [
		"CVE-2015-5143",
		"CVE-2015-5144",
		"UBUNTU-CVE-2015-5143",
		"UBUNTU-CVE-2015-5144"
	],
	"summary": "python-django vulnerabilities",
	"details": "Eric Peterson and Lin Hua Cheng discovered that Django incorrectly handled\nsession records. A remote attacker could use this issue to cause a denial\nof service. (CVE-2015-5143)\n\nSjoerd Job Postmus discovered that DJango incorrectly handled newline\ncharacters when performing validation. A remote attacker could use this\nissue to perform header injection attacks. (CVE-2015-5144)\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:14.04:LTS",
				"name": "python-django",
				"purl": "pkg:deb/ubuntu/python-django?arch=src?distro=trusty"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.6.1-2ubuntu0.9"
						}
					]
				}
			],
			"versions": [
				"1.5.4-1ubuntu1",
				"1.6-1",
				"1.6.1-1",
				"1.6.1-2",
				"1.6.1-2ubuntu0.1",
				"1.6.1-2ubuntu0.2",
				"1.6.1-2ubuntu0.3",
				"1.6.1-2ubuntu0.4",
				"1.6.1-2ubuntu0.5",
				"1.6.1-2ubuntu0.6",
				"1.6.1-2ubuntu0.8"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-django",
						"binary_version": "1.6.1-2ubuntu0.9"
					},
					{
						"binary_name": "python-django-doc",
						"binary_version": "1.6.1-2ubuntu0.9"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-2671-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://ubuntu.com/security/notices/USN-2671-1"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2015-5143"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2015-5144"
		}
	]
}
